traitlets/traitlets.py

```diff
@@ -928,7 +964,7 @@ def unobserve_all(self):
         """Remove all trait change handlers."""
         self._trait_notifiers = {}
 
-    def register_validator(self, handler, name):
+    def _register_validator(self, handler, name):
```

Comment:
Quote @rmorshea "the PR that made this change should already be merged"
Me "Make sure the PR was merged and if it was, rebase so this doesn't show in the diff"